src/pages/RadicalCollaboration/VolunteerCardDesktop.js

```diff
@@ -27,13 +27,13 @@ const VolunteerCardDesktop = (props) => {
                 </Typography>
                 <br></br>
                 <Typography className={classes.dtpLine1}>
-                                    Please <a className={classes.inLineLinkText} href="https://www.hackforla.org/#about">complete this application</a> if you would like
+                                    Please <a className={classes.inLineLinkText} href="https://www.hackforla.org/#about" target="_blank" rel="noopener noreferrer">complete this application</a> if you would like
```

Comment:
If possible, use material UI Link component instead of <a> tag

Reply:
Link is working for internal Links, But for external Links it is not Working